.Rbuildignore

```diff
@@ -20,6 +20,7 @@ data-raw/
 ^docs$
 ^pkgdown$
 
+^_ocwaignore$
```

Comment:
Is there a reason for the `_` instead of `.`?

Reply:
The RStudio file browser only whitelists certain dotfiles for display - e.g., `.gitignore`, `.Rbuildignore` etc. - so you can create a `.ocwaignore` file, but then you can't see it